src/main/java/org/folio/circulation/services/LostItemFeeChargingService.java

```diff
@@ -195,16 +210,17 @@ private Result<List<CreateAccountCommand>> buildAccountsAndActions(
         accountsToCreate.add(lostItemFeeResult);
       }
 
-      if (policy.getDeclareLostProcessingFee().isChargeable()) {
-        log.debug("Charging lost item processing fee");
+      if (policy.getDeclareLostProcessingFee().isChargeable()
+        || (policy.getActualCostFee().isChargeable()
+        && policy.getDeclareLostProcessingFee().isChargeable())) {
```

Comment:
If `policy.getActualCostFee().isChargeable() && policy.getDeclareLostProcessingFee().isChargeable()` is true then `policy.getDeclareLostProcessingFee().isChargeable()` is also true. Do we need to change this `if` at all?

Reply:
The condition changed to initial set, thanks!